watchmal/dataset/dgcnn/dgcnn_dataset.py

```diff
@@ -0,0 +1,71 @@
+import torch_geometric.data as tgd
+
+import h5py
+import numpy as np
+import torch
+
+# WatChMaL imports
+from watchmal.dataset.h5_dataset import H5Dataset
+from watchmal.dataset.pointnet import transformations
+import watchmal.dataset.data_utils as du
+
+
+##...............................................
+
+class DGCNN_dataset(H5Dataset, tgd.Dataset):
+
+    def __init__(self, h5file, geometry_file, is_distributed, k_neighbours, dynamic=True, n_points=4000, use_orientations=True, transforms=None, is_graph=True):
+        super().__init__(h5file, is_distributed)
+        
+        self.n_points = n_points
+        geo_file = np.load(geometry_file, 'r')
+        self.geo_positions = torch.from_numpy(geo_file["position"]).float()
+        self.geo_orientations = torch.from_numpy(geo_file["orientation"]).float()
+        
+        self.f = h5py.File(h5file,'r')
+        h5_labels = self.f["labels"]
+        self.labels = np.array(h5_labels)
+        self.dynamic = dynamic
+        self.k_neighbours = k_neighbours
+        self.use_orientations = use_orientations
+       
+    
+    def load_edges(self, idx, nhits):
+        edge_index = torch.ones([nhits, nhits], dtype=torch.int64)
+        self.edge_index=edge_index.to_sparse()._indices()
+        
+
+    def  __getitem__(self, item):
+
+        data_dict = super().__getitem__(item)
+        hit_pmts = data_dict["data"]["hit_pmts"]
+        hit_charges = data_dict["data"]["hit_charges"]
+        hit_times = data_dict["data"]["hit_times"]
+
+        hit_positions = self.geo_positions[hit_pmts, :]
+        n_hits = min(self.n_points, hit_pmts.shape[0])
+        
+        if self.dynamic:
+            self.n_points = n_hits            #dynamic graph
+        else:
+            self.n_points = 600               #static graph
```

Comment:
This seems to hard-code the n_points as 600 when not `self.dynamic`, but in `__init__` there's a config parameter for n_points. Should that value be used instead?